### PR TITLE
doc: Added Developer convention document.

### PR DESCRIPTION
**Pull request by KvelaGorrrrnio**
Draft for our developer conventions in the frontend. Logging strategy applies to all of our projects, so that is also defined on [Confluence](https://tv2cms.atlassian.net/wiki/spaces/SOF/pages/4348477470/Logging+Strategy).

### CODE REVIEW

**Review by RasmusAlbrektsen — APPROVED** [review]

**Review by andr9528 — COMMENTED** [review]
There are some todo's that i guess we need to decide on before this gets merged?
When/If we create a similar document for Sofie-Server, how much duplication, if any, will there be? 

The Logging section for sure, but if it is only that single one, i think it is fine. If however there ends up being multiple sections duplicated, then moving/making this document somewhere shared by both environment, might be wiser.

Using GitHub wiki as a Git Submodule is one way i personally have keept a large number of .md files grouped, and accesible from the main repository. Doing something similar should allow the document to be shared between diffrent repositories, if we end up needing to do that.

File: DEVELOPER.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+### Naming conventions
+
+**Variables**
+
+_**TODO:** Find out how to handle global settings such as color palettes, sizings, etc..._
+
+**Attribute order**
+
+When writing scss the suggested attribute order is:
```
Comment (LindvedKrvang) [inline DEVELOPER.md:34]: Should we emphasize that the goal here is to have consistency?

**Review by LindvedKrvang — APPROVED** [review]

File: DEVELOPER.md (hunk 1 above)
Comment (KvelaGorrrrnio) [reply DEVELOPER.md]: Yes. I have updated it to reflect that.

**Comment by KvelaGorrrrnio** [comment]
@andr9528 It is a valid concern. Logging strategy is something we want across all of our projects, so it makes sense that we have one source of truth for it. Currently the source of truth is on Confluence.

Regarding the todos: I have updated the one with units for number variables. But this should be a living document that we can alter and add to when the need arise.

**Pull request merged**